src/charm.py

```diff
@@ -177,6 +206,19 @@ def _on_config_changed(self, _: EventBase) -> None:
                         "extra-user-roles": self.extra_user_roles or "",
                     },
                 )
+        if not self.prefix_active and self.prefix:
+            for rel in self.etcd.relations:
+                self.etcd.update_relation_data(
+                    rel.id,
+                    {
+                        "prefix": self.prefix,
+                    },
+                )
+
+        if self.mtls_client_chain:
+            logger.debug(f"Updating mtls client chain to {self.mtls_client_chain}")
+            for rel in self.etcd.relations:
+                self.etcd.set_mtls_chain(rel.id, self.mtls_client_chain)
```

Comment:
Can't we expand the above `if` to something like `if self.prefix and (not self.prefix_active or self.mtls_client_chain):` and merge this in the preceding loop instead of disabling the linter?